index.html

```diff
@@ -157,5 +157,46 @@ <h2>Skills</h2>
         </ul>
       </section>
     </main>
+    <footer class="aboutContact">
+      <p>
+        I'm alwyas ineterested in hearing about new projects, so if you'd 
+        like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/xzbygvyd" method="POST" class="contactForm">
+        <input type="text" name="user_name" placeholder="Full Name" maxlength="30" required>
+        <input type="email" name="user_email" placeholder="Email address" required>
+        <textarea name="user_text" maxlength="500"  required>Enter text here</textarea>
```

Comment:
- [ ] The `text area should have a placeholder, please add it. 
- [ ] At the same time I ask you to delete the text `Enter text here` inside the `text area, this because if someone wants to add text, is required to delete the text before typing, this step could be annoying to users. 🐶

Reply:
but according to the desktop vesion, we haven't touched i yet

Reply:
but according to the desktop vesion, we haven't touched it yet

Reply:
hello @vichuge

I think we haven't touched yet the desktop version of this project we are still working on the mobile version. The desktop version is planned to be done tomorrow

Reply:
Ok so, if the desktop version isn't required at this point pls continue on the next steps for the review.